### PR TITLE
Add more commands to Azure Static Web Apps

### PR DESCRIPTION
**Pull request by aaronamm**
- Add Azure Static Web Apps show and delete command
- Get a deployment token with GitHub CLI

### CODE REVIEW

File: content/cloud-hosting/azure/azure-static-web-apps.md [hunk 1]
```diff
@@ -39,21 +41,21 @@ order: 1
 
 # Create an empty static app
 - As mentioned in this issue (https://github.com/Azure/static-web-apps/issues/446) setting up DevOps is not currently supported.
```
Comment (dmakeroam) [inline content/cloud-hosting/azure/azure-static-web-apps.md]: You can use ARM or Terraform instead of CLI. It supports Azure DevOps already

**Review by dmakeroam — CHANGES_REQUESTED** [review]

File: content/cloud-hosting/azure/azure-static-web-apps.md (hunk 1 above)
Comment (aaronamm) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: What is ARM cpu?

Comment (dmakeroam) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: Azure ARM Template

Comment (aaronamm) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: @dmakeroam  I think this is a simple content, I want to stay with Azure CLI.

Comment (dmakeroam) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: Yes but this sentence is wrong "setting up DevOps is not currently supported."

Comment (aaronamm) [reply content/cloud-hosting/azure/azure-static-web-apps.md]: @dmakeroam  agree, thanks.

**Pull request merged**